packages/web-client/app/components/common/forms/index.css

```diff
@@ -0,0 +1,6 @@
+.form-input-info {
```

Comment:
I feel like this is something that can be reusable so I put it under common. If boxel had a dropdown then I guess it would make sense to add "info text" support there but since it doesn't I think we can have this here? 

![image](https://user-images.githubusercontent.com/273660/150001347-2800f1f9-ba50-4228-97ee-4967f8d3a345.png)